docs/MainSequenceDiagram.puml

```diff
@@ -0,0 +1,45 @@
+@startuml
+actor Doctor
+
+Doctor -> ":Main" ++: initialises the application
+
+":Main" -> ":Scanner"**: new Scanner()
+activate ":Scanner"
+":Main" <-- ":Scanner": in:Scanner
+deactivate ":Scanner"
+
+":Main" -> ":Records"**: new Records()
+activate ":Records"
+":Main" <-- ":Records": records:Records
+deactivate ":Records"
+
+":Main" -> ":AppointmentRecord"**: new AppointmentRecord()
+activate ":AppointmentRecord"
+":Main" <-- ":AppointmentRecord": appointmentRecord:AppointmentRecord
+deactivate ":AppointmentRecord"
+
+":Main" -> ":FileHandler" ++: initFile(records)
+return
+
+":Main" -> ":FileHandler" ++: initFile(appointmentRecord)
+return
+
+":Main" -> ":CommandHandler"**: new CommandHandler()
+activate ":CommandHandler"
+":Main" <-- ":CommandHandler": commandHandler:CommandHandler
+deactivate ":CommandHandler"
+
+":Main" -> ":AppointmentRecord" ++: appointmentNotice()
+return
+
+loop always true
```

Comment:
Instead of always true, I think can write a specific condition e.g. until the exit command is inputted by the user

Reply:
Yes that's a much better way!